pkg/pillar/base/log.go

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) 2020 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package base
+
+import (
+	log "github.com/sirupsen/logrus"
+)
+
+// Debug :
+func (object *LogObject) Debug(args ...interface{}) {
+	if !object.Initialized {
+		log.Errorf("LogObject used without initialization")
```

Comment:
If I get one of these errors I have absolutely no idea where the missing init is. Either they should all be Fatal so I get a stack trace, or they need to log the args so  I can use hat to find the broken code.

Reply:
Will make it Fatal